flink-table/flink-sql-parser/pom.xml

```diff
@@ -252,6 +252,7 @@ under the License.
                      it under ${project.build.directory} where all freemarker templates are. -->
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-dependency-plugin</artifactId>
+				<!-- Uncomment it back once fix for CALCITE-6266 comes to Flink with Calcite upgrade
```

Comment:
I would suggest creating a dedicated Flink jira ticket for the follow-up cleanup task, since not only the pom but also the Parser.jj file need to be reverted.

Reply:
no need for this anymore since Parser.jj is already in Flink